core/templates/services/assets-backend-api.service.ts

```diff
@@ -66,7 +66,8 @@ export class AssetsBackendApiService {
       urlPrefix + '/<entity_type>/<entity_id>/assets/<asset_type>/<filename>');
   }
 
-  async loadAudio(explorationId: string, filename: string): Promise<AudioFile> {
+  async loadAudioAsync(
+      explorationId: string, filename: string): Promise<AudioFile> {
     if (this.isCached(filename)) {
       return new AudioFile(filename, this.assetsCache.get(filename));
```

Comment:
(Question out of the context of this PR, feel free to leave it if you aren't able to understand/fix it.)

If the file is cached then we aren't returning a promise, is it working fine for that case?

Reply:
@srijanreddy98, will it work fine if we won't return a promise here?

Reply:
Yes, the reason being we are using `async` keyword here.